core/src/main/java/bisq/core/CoreModule.java

```diff
@@ -118,15 +84,6 @@ protected void configure() {
         String referralId = environment.getProperty(AppOptionKeys.REFERRAL_ID, String.class, "");
         bind(String.class).annotatedWith(Names.named(AppOptionKeys.REFERRAL_ID)).toInstance(referralId);
 
-        bind(MobileNotificationService.class).in(Singleton.class);
-        bind(MobileMessageEncryption.class).in(Singleton.class);
```

Comment:
MobileMessageEncryption doesn't have singleton annotation and not tested in GuiceSetupTest.java